enhancements/installer/aws-custom-edge-machineset-local-zones.md

```diff
@@ -0,0 +1,933 @@
+- preferably, the node should not receive regular cluster workloads, mainly ingress. due the
+  unsupported network load balancers, latency, and costs.
+
+Presently, the installer does not support Local Zones subnets on day zero operations
+for those reasons:
+- the Local Zones do not support network load balancers (NLB or CLB), used by
+  ingress deployed on the `worker` pool.
+- the Local Zones do not support Nat Gateway, used as a gateway on the default IPI
+  deployment for each subnet and availability zone.
+- the default instance type for compute nodes used on IPI, `m6i.xlarge`, is not
+  available on Local Zones
+- the default EBS type for IPI, `gp3`, is not available in the most Local Zones locations
+- segment the `worker` compute node pool is not allowed
+
+
+### User Stories
+
+Phase 1:
+
+- As a cluster admin, I want the ability to specify a set of subnets on the AWS
+  Local Zones locations to deploy worker nodes, so I can create custom applications
+  to deliver low latency to my end users.
+
+- As a cluster admin, I would like to create a cluster extending worker nodes to the
+  edge of the AWS cloud provider with Local Zones, so I can create custom applications
+  to deliver low latency to my end users.
+
+- As a cluster admin, I would like to control the workloads running on the edge
+  machine pool, and keep it dedicated to applications that are latency sensitive,
+  so I can have a cost-effective infrastructure, avoiding mixing workloads in different locations.
+
+- As a cluster admin, I would like to select existing subnets from the local and
+  the parent region zones, to install a cluster, so I can manage my custom networks with
+  my automation.
+
+- As a developer of VR/AR apps, I would like to use the power of Cloud GPU instances to process
+  images closest to my users, so I can deliver real-time experience.
+
+- As an autonomous application running in an embedded machine, I would like to process the data collected from many
+  sensors and cameras on the Edge of the Cloud with real-time decision-making to my production line.
+
+Phase 2:
+
+- As a cluster admin, I would like to install OpenShift clusters, extending the
+  compute nodes to the Local Zones in my day-zero operations without needing to
+  set up the network and compute dependencies, so I can speed up the edge adoption
+  in my organization using OKD/OCP.
+
+### Goals
+
+Phase 1:
+
+- Admins can create machine pools named `edge` when installing a cluster in existing VPC with subnets in AWS Local Zones.
+- Admins can select and configure subnets on Local Zones before cluster creation.
+- Ensure the installer allows creating a new machine pool for `edge` workloads
+- Ensure the installer can create the MachineSet with `NoSchedule` taints on edge machine pools.
+- Ensure Local Zones subnets will not be used on `worker` machine pools or control planes.
+- Ensure the Wavelength zone will not be used in any machine pool
```

Comment:
How do we distinguish wavelength zones?

Reply:
The [API attribute](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_AvailabilityZone.html)  `zoneType==wavelength-zone`